test/export/test_converter.py

```diff
@@ -571,6 +574,23 @@ def func2(x, y):
             func2(*inp),
         )
 
+    def test_implicit_constant_to_tensor_handling(self):
+        def func1(x):
+            return x + 2
+
+        def func2(x, y):
+            return x * y / (x - 2 * y) + y
+
+        def func3(x):
+            return x + torch.Tensor([3])
```

Comment:
```suggestion
            return x + torch.tensor([3])
```

i believe torch.Tensor is a type?